### PR TITLE
 [MIG] stock_warehouse_orderpoint_stock_info_unreserved: Migration to 12.0

### PR DESCRIPTION
**Pull request by Saran440**

### CODE REVIEW

**Comment by Saran440** [comment]
This module depends on stock_available_unreserved

[OCA/stock-logistics-warehouse #526](https://github.com/OCA/stock-logistics-warehouse/pull/526)

**Comment by kittiu** [comment]
> This module depends on stock_available_unreserved
> 
> [OCA/stock-logistics-warehouse #526](https://github.com/OCA/stock-logistics-warehouse/pull/526)

The dependency is merged.

**Comment by rousseldenis** [comment]
@Saran440 Could you rebase ?

**Comment by Saran440** [comment]
@rousseldenis rebase done.

**Review by tbaden — APPROVED** [review]
code review: lgtm

File: stock_warehouse_orderpoint_stock_info_unreserved/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "version": "12.0.1.0.0",
+    "depends": [
+        "stock_warehouse_orderpoint_stock_info",
+        "stock_available_unreserved"
+    ],
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse",
+    "license": "AGPL-3",
```
Comment (tbaden) [inline stock_warehouse_orderpoint_stock_info_unreserved/__manifest__.py:16]: just noticed, installable: true is missing

**Review by tbaden — CHANGES_REQUESTED** [review]

**Comment by alexschubert** [comment]
Thanks for your contribution!

Functional testing on runbot was unsuccessful due to the issue mentioned by @tbaden .

I will continue testing after this has been resolved.

**Comment by rousseldenis** [comment]
I will stop a 'fake news' : 'installable' key, when omitted is True by default. So, I relaunched runbot and :
image


So, to disable a module, set 'installable' to False

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by tbaden** [comment]
it may be installable, but it just don't feel complete without it in the manifest. maybe just my inner demon.

**Comment by rousseldenis** [comment]
> it may be installable, but it just don't feel complete without it in the manifest. maybe just my inner demon.

I like too implicit things :smile: not blocking one

@tbaden Could you update your review?

**Review by tbaden — APPROVED** [review]

**Comment by Saran440** [comment]
fixed it.

**Pull request merged**